src/containers/FormikForm/VersionActionButtons.tsx

```diff
@@ -23,6 +23,28 @@ interface Props {
   currentLanguage: string;
 }
 
+const StyledResetButton = styled.button`
```

Comment:
_Litt_ misvisende navn i og med at den brukes til to forskjellige ting. Kanskje StyledActionButton eller nokke.